internal/cmd/schema-registry/command_region.go

```diff
@@ -0,0 +1,28 @@
+package schemaregistry
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+)
+
+type schemaRegistryCloudRegion struct {
+	ID          string   `human:"ID" serialized:"id"`
+	Cloud       string   `human:"Cloud" serialized:"cloud"`
+	RegionName  string   `human:"Region Name" serialized:"region_name"`
+	DisplayName string   `human:"Display Name" serialized:"display_name"`
+	Packages    []string `human:"Packages" serialized:"packages"`
```

Comment:
Can we split this into two separate structs?
1. A human-readable struct where this is a comma-delimited string
2. A serialized struct where this is a `[]string`

Reply:
Similar example: https://github.com/confluentinc/cli/blob/460b285118c9d19d2457cbf2b2cf93b1ff05665e/internal/cmd/admin/command_promo_list.go#L16-L26